man/systemd.special.xml

```diff
@@ -667,6 +671,17 @@
           default).</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><filename>wakeup.target</filename></term>
+        <listitem>
+          <para>A special target unit that is started after
+          <filename>suspend.target</filename>,
+          <filename>hibernate.target</filename> and
+          <filename>hybrid-sleep.target</filename>.  This can be used
+          to pull in units and order them after the system wakes up
+          from sleep.</para>
+        </listitem>
```

Comment:
Indeed. So I guess we just need two good examples: a "before" service and "after" service.